master/usage/migrating-pools.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Changing IP pools
+---
+
+## About changing IP pools
+
+When using {{ site.prodname }} IPAM, each workload is assigned an address from the selection of configured IP pools.
+There are a few reasons you may want to change the IP pool being used on a running cluster, including moving to a
+larger CIDR which can accomodate more workloads, or moving off of a CIDR that was used accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to change from one IP pool to another on a running cluster.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Orchestrator support**
+
+While {{ site.prodname }} supports changing IP pools, not all orchestrators do.
+Be sure to consult the documentation for whichever installer you are using to ensure it supports changing it.
+
+For example, in Kubernetes, all three of the following arguments must be equal to, or contain, the Calico IP pool CIDRs:
+
+- kube-apiserver's `--pod-network-cidr`
+- kube-proxy's `--cluster-cidr`
+- kube-controller-manager's `--cluster-cidr`
+
+OpenShift does not support changing the pod network CIDR (as per their [documentation on the `osm_cluster_network_cidr` configuration field](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables).
+
+**Application availability impact**
+
+This process will require the recreation of all {{ site.prodname }}-networked workloads, which will have some impact on
+the availability of your applications.
+
+## Consequences of deleting of an IP pool without following this migration procedure
+
+Removing an IP pool without following the migration procedure can cause network connectivity disruptions in any running
```

Comment:
Just to avoid confusion, I would switch "the" out for "this" (that makes it a little more clear which migration procedure you are referring to)